### DISCUSSION TITLE
override selection for model

### DISCUSSION THREAD

**Discussion by zaxarhic2134** [aminalaee/sqladmin#716] [Q&A]
I have a problem with select overriding in SelectMultipleField, I have a model Variable:

    class Variable(Base):
        tablename = 'variable'
        id = Column(Integer, primary_key=True)
        connection_id = Column(Integer, ForeignKey('connection.id'))  # предполагается наличие связи
        variables = Column(ARRAY(String))
        hierarchy = Column(JSON)
        selected_variables = Column(ARRAY(String))
        selected_hierarchy = Column(JSON)
        connection = relationship("Connection", back_populates="variable")

And the model for sqladmin:
```
class VariableAdmin(ModelView, model=Variable):
    column_list = ['id', 'connection_id', 'variables', 'selected_variables']
```

If you do not override the Variables field, SelectMultipleField will be implemented

![image](https://github.com/aminalaee/sqladmin/assets/91330925/23cccfe2-3da9-4c7f-9538-f0ec5e0b3231)

And I would like to make a similar selection for the selected_variables field from the existing Variables list

![image](https://github.com/aminalaee/sqladmin/assets/91330925/a760e726-06ff-4dee-8d59-cde48b1ca9c7)

I tried to add a Varialbes selection via form_args but it didn't work, how can I implement this and if possible how can I handle this selection so that the selected_hierarchy field displays data depending on the selection

**Comment by mmzeynalli** [comment]
Have you tried `form_ajax_refs`?
